build.gradle

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Aiven Oy
+ * Copyright 2022 Aiven Oy and jdbc-connector-for-apache-kafka contributors
```

Comment:
year 2021 should stay or alternatively:
`Copyright 2021 - 2022 Aiven Oy and jdbc-connector-for-apache-kafka contributors`.

Reason for this is that file was created in 2021, copyright year should be either the earlier year, or the year range from first change until last change.